great-idea-website/index.html

```diff
@@ -15,49 +15,87 @@
 </head>
 
   <body>
-
-  Services
-  Product
-  Vision
-  Features
-  About
-  Contact
-
-  <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
-  Innovation
-  On
-  Demand
-
-  Get Started
-
-  <img src="img/header-img.png" alt="Image of a code snippet.">
-
-  Features
+    <header>
+    <div class="container">
```

Comment:
Over here you have a header tag as well as in inner div.  I would say one of them is redundant as it doesn't provide any additional functionality.

Also, conventionally the container class is used to targeting the div/body that is wrapping the page and not the header.  I would name the container class something else, like navbar.